### PR TITLE
4919 queries measurement, 4931 token delays

### LINKED ISSUE #4919: Document the InfluxDB IOx queries measurement
When using an external visualisation tool (such as Grafana), a call to list available tables/measurements will include one from the `system` namespace called `queries`.

This table shows queries which have been executed  for that namespace on the querier node answering the query against .`queries`

This may lead to confusion as users will see a measurement they did not create and/or may not understand that the response will not necessarily include all of their queries.

Related to https://github.com/influxdata/influxdb_iox/issues/7119

### LINKED ISSUE #4931: Delay between token creation and token being active for queries
With Cloud Dedicated, a (read) token might not be usable for querying immediately after the token is created. Tokens can take seconds or minutes to become active for querying.

##### Relevant URLs
https://github.com/influxdata/idpe/issues/17578

### PR DESCRIPTION
**Pull request by jstirnaman**
Closes #4919
Closes #4931 

- [ ] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud-dedicated/admin/tokens/create.md [hunk 1]
```diff
@@ -38,6 +38,8 @@ influxctl token create \
 The command returns the token ID and the token string.
 **This is the only time the token string is available in plain text.**
 
+InfluxDB might take some time to synchronize new tokens. If a new token doesn't immediately work for querying or writing, wait a few seconds or minutes and then retry your request.
```
Comment (sanderson) [inline content/influxdb/cloud-dedicated/admin/tokens/create.md:41]:
```suggestion
InfluxDB might take some time to synchronize new tokens.
If a new token doesn't immediately work for querying or writing, wait a few seconds and try again.
```

File: content/influxdb/cloud-dedicated/reference/internals/system-tables.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+### queries measurement schema
+
+- **system.queries** _(measurement)_
+  - **fields**:
+      - **issue_time**: timestamp when the query was issued
+      - **query_type**: type (syntax: `sql`, `flightsql`, or `influxql`) of the query
+      - **query_text**: query statement text
+      - **success**: execution status (boolean) of the query
+      - **completed_duration**: time (duration) that the query took to complete
+      - **trace_id**: trace ID for debugging and monitoring events
```
Comment (sanderson) [inline content/influxdb/cloud-dedicated/reference/internals/system-tables.md:55]: Maybe add the field types on these (string, float, int, etc.)

File: content/influxdb/cloud-dedicated/reference/internals/system-tables.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+`system.queries` reflects a process-local, in-memory, namespace-scoped query log.
+While this table may be useful for debugging and monitoring queries, keep the following in mind:
+
+- Query entries stored in `system.queries` are volatile.
+  - Records are lost on pod restarts.
+  - Queries for one namespace can evict records from another namespace.
+- Data reflects the state of a specific pod answering queries for the namespace.
+  - A query for records in `system.queries` can return different results depending on the pod the request was routed to.
+
+**Data retention:** Data can be transient and is deleted on pod restarts.
```
Comment (sanderson) [inline content/influxdb/cloud-dedicated/reference/internals/system-tables.md:44]:
```suggestion
**Data retention:** System data can be transient and is deleted on pod restarts.
```

File: content/influxdb/cloud-serverless/reference/internals/system-tables.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
+---
+title: InfluxDB system tables
+description: >
+  InfluxDB system measurements contain time series data used by and generated from the
+  InfluxDB monitoring system.
```
Comment (sanderson) [inline content/influxdb/cloud-serverless/reference/internals/system-tables.md:5]:
```suggestion
  InfluxDB internal monitoring system.
```

File: content/influxdb/cloud-dedicated/reference/internals/system-tables.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
+---
+title: InfluxDB system tables
+description: >
+  InfluxDB system measurements contain time series data used by and generated from the
+  InfluxDB monitoring system.
```
Comment (sanderson) [inline content/influxdb/cloud-dedicated/reference/internals/system-tables.md:5]:
```suggestion
  InfluxDB internal monitoring system.
```

File: content/influxdb/cloud-serverless/reference/internals/system-tables.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+    name: System tables
+    parent: InfluxDB Cloud internals
+weight: 103
+influxdb/cloud-serverless/tags: [tables, information schema]
+related:
+  - /influxdb/cloud-serverless/reference/sql/information-schema/
+---
+
+InfluxDB system measurements contain time series data used by and generated from the
+InfluxDB monitoring system.
```
Comment (sanderson) [inline content/influxdb/cloud-serverless/reference/internals/system-tables.md:17]:
```suggestion
InfluxDB internal monitoring system.
```

File: content/influxdb/cloud-serverless/reference/internals/system-tables.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+Each InfluxDB Serverless namespace includes the following system measurements:
+
+- [queries](#_queries-system-measurement)
```
Comment (sanderson) [inline content/influxdb/cloud-serverless/reference/internals/system-tables.md:21]:
```suggestion
- [queries](#queries-system-measurement)
```

File: content/influxdb/cloud-serverless/reference/internals/system-tables.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+### queries measurement schema
+
+- **system.queries** _(measurement)_
+  - **fields**:
+      - **issue_time**: timestamp when the query was issued
+      - **query_type**: type (syntax: `sql`, `flightsql`) of the query
+      - **query_text**: query statement text
+      - **success**: execution status (boolean) of the query
+      - **completed_duration**: time (duration) that the query took to complete
+      - **trace_id**: trace ID for debugging and monitoring events
```
Comment (sanderson) [inline content/influxdb/cloud-serverless/reference/internals/system-tables.md:55]: Same comment as above. Might be helpful to add field types.

**Review by sanderson — APPROVED** [review]
Some minor suggestions. Looks really good.

File: content/influxdb/cloud-dedicated/reference/internals/system-tables.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+```sql
+SELECT issue_time, query_type, query_text, success FROM system.queries;
+```
+
+_When listing measurements (tables) available within a namespace, some clients and query tools may include the `queries` table in the list of namespace tables._
+
+`system.queries` reflects a process-local, in-memory, namespace-scoped query log.
+While this table may be useful for debugging and monitoring queries, keep the following in mind:
+
+- Query entries stored in `system.queries` are volatile.
```
Comment (btasker) [inline content/influxdb/cloud-dedicated/reference/internals/system-tables.md:38]:
```suggestion
- Records stored in `system.queries` are volatile.
```

File: content/influxdb/cloud-dedicated/reference/internals/system-tables.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+    name: System tables
+    parent: InfluxDB internals
+weight: 103
+influxdb/cloud-dedicated/tags: [tables, information schema]
+related:
+  - /influxdb/cloud-dedicated/reference/sql/information-schema/
+---
+
+InfluxDB system measurements contain time series data used by and generated from the
+InfluxDB monitoring system.
```
Comment (btasker) [inline content/influxdb/cloud-dedicated/reference/internals/system-tables.md:17]:
```suggestion
InfluxDB internal monitoring system.
```

**Review by btasker — APPROVED** [review]
A couple  of very minor suggestions, but LGTM - thanks!

**Pull request merged**